addons/account/models/company.py

```diff
@@ -231,6 +231,26 @@ class ResCompany(models.Model):
     # Autopost Wizard
     autopost_bills = fields.Boolean(string='Auto-validate bills', default=True)
 
+    # Tax ex/included in prices
+    account_price_include = fields.Selection(
+        selection=[('tax_included', 'Tax Included'), ('tax_excluded', 'Tax Excluded')],
+        string='Default Sales Price Include',
+        default='tax_excluded',
+        required=True,
+        help="Default on whether the sales price used on the product and invoices with this Company includes its taxes."
+    )
+
+    @api.constrains("account_price_include")
+    def _check_set_account_price_include(self):
+        if any(company.sudo()._existing_accounting() for company in self):
+            raise ValidationError("Cannot change Price Tax computation method on a company that has already started invoicing.")
+
+    def _reset_default_price_include(self):
+        if not self._existing_accounting():
+            new_value = self._get_default_account_price_include()
+            if new_value != self.account_price_include:
+                self.account_price_include = new_value
```

Comment:
This is actually never called now. I can make a new PR to remove it in case.

Reply:
Followup draft PR: https://github.com/odoo-dev/odoo/pull/new/master-account-pricepocalypse-followup-pgi